docs/index.rst

```diff
@@ -191,37 +191,53 @@ and change the cache location. Caching persists between Astroquery sessions.
 If you know the service you are using has released new data recently, or if you believe you are
 not recieving the newest data, try clearing the cache.
 
+Service specific settings
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The Astroquery cache location is divided by service, so each service's cache should be managed invidually,
-however whether the cache is active and the expiration time are controlled centrally through the
-astroquery ``cache_conf`` module. Astroquery uses the Astropy configuration infrastructure, information about
+The Astroquery cache location is specific to individual services,
+so each service's cache should be managed invidually.
+The cache location can be viewed with the following command
+(using `Simbad <simbad/simbad.rst>`_ as an example):
+
+.. code-block:: python
+
+    >>> from astroquery.simbad import Simbad
+    >>> print(Simbad.cache_location)   # doctest: +IGNORE_OUTPUT
+    /Users/username/.astropy/cache/astroquery/Simbad
+
+Each service's cache is cleared with the `clear_cache` function within that service.
+
+.. code-block:: python
+
+    >>> import os
+    >>> from astroquery.simbad import Simbad
+    ...
+    >>> os.listdir(Simbad.cache_location)   # doctest: +IGNORE_OUTPUT
+    ['8abafe54f49661237bdbc2707179df53b6ee0d74ca6b7679c0e4fac0.pickle',
+    '0e4766a7673ddfa4adaee2cfa27a924ed906badbfae8cc4a4a04256c.pickle']
+    >>> Simbad.clear_cache()
+    >>> os.listdir(Simbad.cache_location)   # doctest: +IGNORE_OUTPUT
+    []
+
+Astroquery-wide settings
```

Comment:
maybe also add a line for deleting all astroquery cache. I just brute force it myself, but a nice example may be useful for the users :)

Reply:
Is there a way to not brute force it?

Reply:
I don't know, you're the cache expert :)
(as I just keep deleting everything from bash as I don't expect to have anything valuable in the cache)

Reply:
OK, then there isn't another way. Something to consider adding then. 😄